src/utils/__tests__/style-formatter.spec.js

```diff
@@ -134,5 +136,32 @@ describe('style-formatter', () => {
       const formattedStyle = styleFormatter.getStyles(style, disabled, Theme);
       expect(formattedStyle.includes('opacity: 0.4')).to.be.false;
     });
+    it('should return font-weight: bold when bold is selected', () => {
+      style = {
+        textStyle: {
+          bold: true,
+        },
+      };
+      const formattedStyle = styleFormatter.getStyles(style, disabled, Theme);
+      expect(formattedStyle.includes('font-weight: bold')).to.be.true;
+    });
+    it('should return font-style: italic when italic is selected', () => {
+      style = {
+        textStyle: {
+          italic: true,
+        },
+      };
+      const formattedStyle = styleFormatter.getStyles(style, disabled, Theme);
+      expect(formattedStyle.includes('font-style: italic')).to.be.true;
+    });
+    it('should return text-decoration: underline when underline is selected', () => {
+      style = {
+        textStyle: {
+          underline: true,
+        },
+      };
```

Comment:
In the PR for borders I changed the setup a bit for the unit test. style is reset in a beforeEach. Depending on which one we merge first there are some changes needed here. Setting the specific object on style instead of setting the whole style object